models/userroom.js

```diff
@@ -1,31 +1,35 @@
 Chatter.UserRoom = ChatterUserRoom = Astro.Class({
```

Comment:
What is a UserRoom as opposed to a Room? Please make a doc block at the top of the file to explain